awx/main/analytics/core.py

```diff
@@ -97,6 +97,9 @@ def gather(dest=None, module=None, collection_type='scheduled'):
         from awx.main.analytics import collectors
         module = collectors
 
+    tgz_tempdir = getattr(settings, 'ANALYTICS_GATHER_DIR', None)
```

Comment:
Whenever there's a new setting introduced like this, we generally also put a matching default definition somewhere in `awx/settings/defaults.py` e.g.,

```python
settings.ANALYTICS_GATHER_DIR = None
```

Reply:
Also, cc @gamuniz you're making a new directory here, but it's not actually being used below?

Reply:
@ryanpetrello thanks for your comments. PR updated